src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -196,8 +196,13 @@ public function testTranslator()
             '->registerTranslatorConfiguration() finds Form translation resources'
         );
         $ref = new \ReflectionClass('Symfony\Component\Security\Core\SecurityContext');
+        $ref = dirname($ref->getFileName());
+        if (!file_exists($ref.'/composer.json')) {
+            $this->assertFalse(file_exists($ref.'/Resources/translations/'));
```

Comment:
This test should rather be in the security component (otherwise you cannot remove this in 2.7 because of the 2.7 low deps).

And then, in 2.7+, it should be replaced by a test ensuring they are in sync (or at least than the new files don't miss any translations from the old file).